ssl/s3_lib.c

```diff
@@ -4514,17 +4476,20 @@ const SSL_CIPHER *ssl3_choose_cipher(SSL_CONNECTION *s, STACK_OF(SSL_CIPHER) *cl
 
 int ssl3_get_req_cert_type(SSL_CONNECTION *s, WPACKET *pkt)
 {
-    uint32_t alg_k, alg_a = 0;
```

Comment:
I know its not your doing here @kroeckx , but all the functions in the s3_lib.c don't really explicitly relate to ssl3.  Is it worth renaming them with a tls prefix, just to make their use clear?   It feels odd to me to be removing SSLv3 and leaving so many functions prefixed with ssl3 in place.